R/create_package.R

```diff
@@ -176,9 +182,14 @@ RoxygenNote: %5$s
 
   # add LICENSE.md
   file.copy(
-    system.file(
-      file.path("generic_template", "gplv3.md"),
-      package = "checklist"
+    switch(
+      license,
+      "GPL-3" = system.file(
+        file.path("generic_template", "gplv3.md"), package = "checklist"
+      ),
+      "MIT" = system.file(
+        file.path("generic_template", "mit.md"), package = "checklist"
```

Comment:
Fill in `<YEAR>` with the current year and `<COPYRIGHTHOLDER>` with Research Institute for Nature and Forest.

Reply:
Done in [9bc3779](https://github.com/inbo/checklist/pull/92/commits/9bc3779f77df8ff73c358d6252dc313457334568)